source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing_helper.py

```diff
@@ -93,6 +74,7 @@ def build_ridesharing_journeys(from_pt_obj, to_pt_obj, period_extremity, instanc
     try:
         rsjs = instance.get_ridesharing_journeys(from_str, to_str, period_extremity)
     except: # TODO handle it smarter
+        logging.exception('')
```

Comment:
hmm, stupid question: what does this line log? nothing? the stack?

Reply:
logs the exception (so stack and exception object)